crates/bevy_pbr/src/bloom/bloom.wgsl

```diff
@@ -0,0 +1,131 @@
+#import bevy_core_pipeline::fullscreen_vertex_shader
+
+struct Uniforms {
+	threshold: f32,
+	knee: f32,
+	scale: f32,
+};
+
+@group(0) @binding(0) var org: texture_2d<f32>;
```

Comment:
Small nitpick, but most bevy shaders put the attributes on their own line. Would be nice if we had a wgslfmt to standardize formatting.

Reply:
Ngl I kinda hate the separate line thing, it bloats the file so much and makes it harder to read. But I can change it to match.

Reply:
Yeah, I don't really mind either way, I just want everything standardized automatically like all the rust file and never have to think about it.